SPCTR-3467: Fix - Icon List vertical alignment issue on mobile devices in editor (#3579)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Global - Resolved an issue in which the Spectra blocks were unable to be previewed in the widget editor customizer.
* Fix: Buttons - Resolved an issue in which the margins added to the button group would get overridden by child button margins.
* Fix: Table Of Contents - Resolved an issue wherein the Table Of Contents collapse feature would not work as intended.	

> * Fix: Icon List - Resolved an issue where the vertical alignment breaks in editor on mobile devices.

Reviewer: ```suggestion
* Fix: Icon List - Resolved an issue wherein the vertical alignment would not appear as intended in the editor mobile view.
```

Author: Done.
